### ISSUE TITLE
After update to 8.1 contact birthdays are not shown in calendar anymore

### ISSUE THREAD

**Issue by smfzutzg** [owncloud/contacts#976]
After update to 8.1 contact birthdays are not shown in calendar anymore

**Comment by irgendwie** [comment]
I have a similar problem:

Birthdays are still shown in the calendar app, but I'm not able to synchronize them via CalDav anymore. I didn't change any settings in my CalDav client since owncloud vers. 8.

owncloud vers: git, stable8.1 branch
contacts app vers: git, stable8.1 branch
calendar app vers: git, stable8.1 branch

**Comment by RealRancor** [comment]
Dup of: https://github.com/owncloud/calendar/issues/824, got fixed with calendar 0.7.1 and the next release of the contacts app with https://github.com/owncloud/contacts/pull/970

**Comment by DeepDiver1975** [comment]
As far as I can tell there is an issue with caldav - birthdays never show up for me - no clue why

**Comment by MeCias** [comment]
> Dup of: owncloud/calendar#824, got fixed with calendar 0.7.1 and the next release of the contacts app with #970

Couldn't get it to work in 0.7.1, but I can confirm that since 0.7.2 the birthdays are synchronized again between my Thunderbird or my Android phone.

THX for that

**Comment by ainola** [comment]
Potentially related to #986?

**Comment by RealRancor** [comment]
Not sure but definitely related to the already linkef https://github.com/owncloud/contacts/pull/970 which is not yet released as a new version of the contacts app.

**Comment by pruje** [comment]
Bug confirmed on 2 ownCloud instances:
- 7.0.4 upgraded to 8.1.1
- 8.1.1 fresh install
The 2 instances uses contacts v0.4.0 and calendar v0.7.3

**Comment by gvde** [comment]
With #970?

**Comment by FloEdelmann** [comment]
Bug confirmed with ownCloud 8.1.1, Calendar 0.7.3 and Contacts 0.4.0.0 with https://github.com/owncloud/contacts/commit/990a915a19cdc5bdd45e1a193a478d360eeaddc2

**Comment by gvde** [comment]
Fill out the template from the contribution guidelines...

**Comment by RealRancor** [comment]
@pruje 
> 7.0.4 upgraded to 8.1.1

Hopefully you havn't directly upgraded from 7.0.4 to 8.1.1? This is not supported at you need to go to 8.0.x first.

**Comment by pruje** [comment]
@RealRancor 
Sorry, 8.0.4 not 7.0.4

**Comment by pruje** [comment]
@gvde Yes, fix #970 works for me. I'm not sure to understand your last post; do I need to paste here full details of my ownCloud instances, formatted like the issue template?
Sorry, I'm new here, many thanks for your help.

**Comment by RealRancor** [comment]
@pruje 

Yes, when reporting issues (no matter if in core or in such apps repositories like contacts) you should always use the issue template. Copy and past the template into your issue and fill out all relevant data into it.

**Comment by gvde** [comment]
@pruje Now it's fully unclear to me whether or not you still have a problem. First you have "confirmed" the bug. Now you write #970 "works for" you. Does that mean #970 fixes the problem for you? Or do you still have another problem?

**Comment by pruje** [comment]
@gvde Sorry for the misunderstandings. Basically, I just wanted to report that this current issue was still open on the latest versions, despite the fact #970 & #971 were merged since july 9th, and knowing that ownCloud 8.1.1 was released august 11th. The tag 'needs info' suggested me that it has to be reported.
When I saw your post about #970, I've read it, did some research and harcoded in my instances the fix (commit dff63bf3bdcca1c0aa38268738c51804380f3f86), and I can confirm to you that this fix actually works. It just needs to be merged for the next ownCloud release, if possible.

**Comment by RealRancor** [comment]
@pruje oC itself is not shipping the contacts app since 8.0 so already merged stuff is not released yet as discussed in various places at this tracker. There is an open issue available at this tracker but no one with the permissions at the appstore was jumping in yet.

**Comment by pruje** [comment]
@RealRancor OK thanks for the info

**Comment by h-2** [comment]
Whats the state on this? Is it fixed for the next release and is there an ETA or should I patch myself? Thank you!

**Comment by RealRancor** [comment]
You need to patch it yourself as it seems no one cares to release a new version of the contacts app.
